include/nbl/builtin/hlsl/subgroup2/arithmetic_portability.hlsl

```diff
@@ -19,24 +19,26 @@ namespace hlsl
 namespace subgroup2
 {
 
-template<typename Config, class BinOp, int32_t _ItemsPerInvocation=1, class device_capabilities=void NBL_PRIMARY_REQUIRES(subgroup::is_configuration_v<Config>)
+template<typename Config, class BinOp, int32_t _ItemsPerInvocation=1, class device_capabilities=void, bool OverrideUseNativeInstrinsics=true NBL_PRIMARY_REQUIRES(is_configuration_v<Config>)
```

Comment:
no, `OverrideUseNativeInstrinsics` has no right to be here, just use `device_capabilities=void` to achieve that.